### PR TITLE
Implement partially transparent visualizations

### LINKED ISSUE #11534: Graph Editor Look&Feel - Slight opacity in the visualization panel if not selected.

### PR DESCRIPTION
**Pull request by vitvakatu**
### Pull Request Description

Closes #11534 

- Visualizations are partially transparent by default
- The initial z-order is undetermined at the project load
- Node is moved on top of other if it is dragged (before it only happened if it was **selected**, which is not the same)
- Changed rendering for edges slightly, to avoid visible edge ends underneath visualization. The implementation of additional offsets is rather naive, but it works.

https://github.com/user-attachments/assets/fba44816-eed9-471d-83a7-8fe6e5892477

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.
- [x] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

File: app/gui/src/project-view/components/GraphEditor/GraphNode.vue [hunk 1]
```diff
@@ -292,6 +292,9 @@ const dragPointer = usePointer(
   { pointerCapturedBy: 'target' },
 )
 const isDragged = computed(() => dragPointer.dragging && significantMove.value)
+watch(isDragged, (newVal, oldVal) => {
+  if (newVal !== oldVal) graph.db.moveNodeToTop(nodeId.value)
+})
```
Comment (farmaazon) [inline app/gui/src/project-view/components/GraphEditor/GraphNode.vue]: AFAIK there is no need for this check, as watch is run only when value changed.

**Comment by kazcw** [comment]
The edges don't exactly meet the visualization:

imagem

We can render it correctly by enabling the `maskSource` for edges coming from nodes with visualizations. @Frizi brought up possibly switching from masking to clipping to improve performance; this would take some geometry, as we'd need to compute a shape whose *inside* includes the part of the line not to occlude.

**Comment by vitvakatu** [comment]
After some internal discussions and a PoC implementation, we found that any obvious fix for imperfections noted by @kazcw above ↑ will lead to a performance drop compared with the current develop.
That is unfortunate, but we will merge this PR and work on a proper solution later.

See #11622 and #11623.

**Review by farmaazon — APPROVED** [review]
I would add a CHANGELOG entry - even if it's minor, it still a UX change.

**Pull request merged**